### PR TITLE
enable subset analysis for topk

### PR DESCRIPTION
**Pull request by lingwei-gu**
enable subset analysis for topk

### CODE REVIEW

**Comment by lintool** [comment]
@ronakice you got this?

**Comment by ronakice** [comment]
Yup!

File: pyserini/eval/evaluate_qa_overlap_retrieval.py [hunk 1]
```diff
@@ -0,0 +1,329 @@
...
+import json
+import logging
+import re
+import unicodedata
+from tqdm import tqdm
+import numpy as np
+import os
+import wget
+import regex
+import collections
```
Comment (ronakice) [inline pyserini/eval/evaluate_qa_overlap_retrieval.py:32]: question, are all the dependencies already part of pyserini's?

Comment (lingwei-gu) [reply pyserini/eval/evaluate_qa_overlap_retrieval.py:32]: everything was in evaluate_dpr_retrival.py except os and wget.
maybe need to add wget in requirement.txt?

Comment (ronakice) [reply pyserini/eval/evaluate_qa_overlap_retrieval.py:32]: Yup, add it, please, unless you see some other workaround without.

**Review by ronakice — APPROVED** [review]
LGTM!

**Pull request merged**